### PR TITLE
Make autonomous_vehicle graph more complex

### PR DESCRIPTION
**Pull request by alycialee**
Add more nodes and topics to mapping, motion planning, and perception. Also add more faults.

### CODE REVIEW

File: graph_generator/config/autonomous_vehicle/mapping.yaml [hunk 1]
```diff
@@ -88,10 +88,158 @@ nodes:
...
         publish:
           - topic: road_data
             value_range: [20, 30]
             delay_range: [1, 2]
+
+  - name: localization
+    subscribe:
+      - topic: gps1
+        valid_range: [0, 10]
+        watchdog: 10
```
Comment (yundddd) [inline graph_generator/config/autonomous_vehicle/mapping.yaml:122]: this is never satisfied. gps1 and gps2 are published periodically every 20 time units.

File: graph_generator/config/autonomous_vehicle/perception.yaml [hunk 2]
```diff
@@ -166,3 +211,276 @@ nodes:
...
+        valid_range: [0, 10]
+        watchdog: 25
+        invalid_input_callback:
+          fault:
+            affect_publish:
+              topic: motion_estimation_data
+              drop: 3
+
+    loop:
+      period: 25
```
Comment (yundddd) [inline graph_generator/config/autonomous_vehicle/perception.yaml:423]: this is too slow when subscriber watchdogs

Comment (yundddd) [inline graph_generator/config/autonomous_vehicle/perception.yaml:423]:
```suggestion
      period: 10
```

File: graph_generator/config/autonomous_vehicle/mapping.yaml (hunk 1 above)
Comment (yundddd) [inline graph_generator/config/autonomous_vehicle/mapping.yaml:122]:
```suggestion
        watchdog: 25
```

File: graph_generator/config/autonomous_vehicle/mapping.yaml [hunk 3]
```diff
@@ -88,10 +88,158 @@ nodes:
...
+        invalid_input_callback:
+          fault:
+            affect_publish:
+              topic: localization_data
+              value: 50
+              count: 2
+
+      - topic: gps2
+        valid_range: [0, 10]
+        watchdog: 10
```
Comment (yundddd) [inline graph_generator/config/autonomous_vehicle/mapping.yaml:132]:
```suggestion
        watchdog: 25
```

File: graph_generator/config/autonomous_vehicle/perception.yaml (hunk 2 above)
Comment (alycialee) [reply graph_generator/config/autonomous_vehicle/perception.yaml]: done

File: graph_generator/config/autonomous_vehicle/mapping.yaml (hunk 3 above)
Comment (alycialee) [reply graph_generator/config/autonomous_vehicle/mapping.yaml]: done

**Review by yundddd — APPROVED** [review]

**Pull request merged**